spec/fabricators/reviews_fabricator.rb

```diff
@@ -0,0 +1,6 @@
+Fabricator(:review) do 
+  video_id {Fabricate(:video).id}
+  user_id {Fabricate(:user).id}
```

Comment:
I define the review fabricator before I wrote the rspec code, so I didn't know I will use it or not, I wrote all the columns is because I think that the Fabricator is to mimic the real object. if this is not good,  so I should define the columns only when I use it?